internal/api/v2/execute_test.go

```diff
@@ -11,9 +11,13 @@ import (
 	"testing"
 	"time"
 
+	. "github.com/AccumulateNetwork/accumulate/internal/api/v2"
+	mock_api "github.com/AccumulateNetwork/accumulate/internal/mock/api"
```

Comment:
snake case?

Sorry, I had to comment on something... :)

Reply:
The Go team's [recommendations on package names](https://go.dev/blog/package-names) state that package names should not include underscores or mixed case. However, `mock_api` is generated code for testing, so :shrug: I'm not going to worry about it.